Cargo.toml

```diff
@@ -73,6 +73,11 @@ wabt = "0.10.0"
 wasm-bindgen-test = "0.3.24"
 which = "4.4.0"
 
+# Light client support:
+smoldot-light = { git = "https://github.com/smol-dot/smoldot.git", default-features = false }
```

Comment:
do we want to `pin a commit rev here`? I reckon it smoldot can have a breaking changes etc, I guess will also prevent us from publishing subxt on crates.io?

Perhaps the idea is to make a wrapper crate with smoldot to begin with?

Reply:
Hmm yeah, we need to reference something on crates.io really.